addons/web/static/src/js/core/utils.js

```diff
@@ -1002,6 +1002,18 @@ var utils = {
             ['name', '=like', '%.assets\_%.js'],
         ];
     },
+    /**
+     * Detects if 2 elements are colliding
+     *
+     * @param {Element} el1
+     * @param {Element} el2
+     * @returns {boolean}
+     */
+    isColliding(el1, el2) {
+        const el1Rect = el1.getBoundingClientRect();
+        const el2Rect = el2.getBoundingClientRect();
+        return !(el1Rect.bottom < el2Rect.top || el1Rect.top > el2Rect.bottom || el1Rect.right < el2Rect.left || el1Rect.left > el2Rect.right);
+    },
```

Comment:
Mmh, why not putting this in web utils indeed but in that case, put it in the DOM utils (dom.js), not this one. Also, the method should be named "areColliding" then